src/config.ts

```diff
@@ -106,6 +106,9 @@ export default {
       'fellowship',
       'basically',
       'long-time-no-see',
+      'superman',
```

Comment:
I think these should match the rules in https://github.com/retextjs/retext-equality/blob/main/rules.md

Reply:
So should I add hero-heroine, superman-superwoman to the list?

Reply:
Yes, please add to the list.